tests/test_synced_attrdict.py

```diff
@@ -514,12 +514,10 @@ def test_nested_types_dict_conversion(self):
         """Ensure that calling methods like items and values does not
         change the type of nested dictionaries."""
         sad = self.get_sad({'a': {'b': 1}})
-        assert type(sad['a']) is SAD
+        assert isinstance(sad['a'], SAD)
```

Comment:
This is actually not the exact same test, so please revert that change here and below.